test/lib/runtime/broker_test.exs

```diff
@@ -3953,7 +3962,7 @@ defmodule Ask.BrokerTest do
     :ok = channel_status_server |> GenServer.stop
   end
 
-  def create_running_survey_with_channel_and_respondent(steps \\ @dummy_steps, mode \\ "sms") do
+  defp create_running_survey_with_channel_and_respondent(steps \\ @dummy_steps, mode \\ "sms") do
```

Comment:
Don't we want to share the function between the two test suites instead of create a copy for each of them?